src/amg/azext_amg/tests/latest/test_amg_livescenario.py

```diff
@@ -18,7 +18,7 @@ class AmgLiveScenarioTest(LiveScenarioTest):
 
     @AllowLargeResponse(size_kb=3072)
     @ResourceGroupPreparer(name_prefix='cli_test_amg', location='westeurope')
-    def test_amg_e2e(self, resource_group):
+    def xxx_test_amg_e2e(self, resource_group):
```

Comment:
It looks like a strange test name.

Reply:
Good catch. This file is meant to be deleted as I have consolidated the e2e test in a single file. Let me remove it right now